apps/examples/smartfs/smartfs_powercut/Makefile

```diff
@@ -54,15 +54,15 @@
 -include $(TOPDIR)/Make.defs
 include $(APPDIR)/Make.defs
 
-# smartfs powercut Test built-in application info
+# jSmartFs powercut Test built-in application info
 
-APPNAME = smartfs_powercut
+APPNAME = jsmartfs_powercut
```

Comment:
The app name needs to be changed for buffer optimization example as well please.
from "smartfs_buf_opt" -> "jsmartfs_buf_opt"